tests/virt_autotest/libvirt_host_bridge_virtual_network.pm

```diff
@@ -66,6 +66,8 @@ sub run_test {
         test_network_interface($guest, mac => $mac, gate => $gate, net => $net);
 
         assert_script_run("virsh detach-interface $guest bridge --mac $mac $exclusive");
+        my $check = script_run("ssh root\@$guest ip l | grep " . $mac, 60);
+        die "Failed to detach bridge interface for guest $guest." if ($check eq 0);
```

Comment:
One liner is enough
```
die "Failed to detach bridge interface for guest $guest." if (script_run("ssh root\@$guest ip l | grep " . $mac, 60) == 0);
```
I think

Reply:
The check looks good to me.  One liner should work the same. :)